### PR TITLE
Only build aarch64 wheels for cpython manylinux

### PR DESCRIPTION
**Pull request by erikjohnston**
As they are too slow to build for everything else

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -318,7 +318,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.cibuildwheel]
 # Skip unsupported platforms (by us or by Rust).
-skip = "cp36* *-musllinux_i686"
+skip = "cp36* *-musllinux_i686 pp*aarch64 musllinux_aarch64"
```
Comment (DMRobertson) [inline pyproject.toml:321]: Should this be `*-musllinux_aarch64`?

**Review by DMRobertson — COMMENTED** [review]
I think this complements #14253 --- we should hopefully only build abi3 wheels for arm64 too?

File: pyproject.toml (hunk 1 above)
Comment (erikjohnston) [reply pyproject.toml:321]: D'oh!

**Comment by erikjohnston** [comment]
> I think this complements #14253 --- we should hopefully only build abi3 wheels for arm64 too?

I think that should happen automatically

**Review by DMRobertson — APPROVED** [review]
As Patrick says elsewhere, I think this was meant to be targeted at 1.70? Otherwise LGTM (but we should check which wheels get built).

And I think:

Resolves #14234.
Resolves #14233.

**Pull request merged**